sdks/python/apache_beam/io/iobase.py

```diff
@@ -890,6 +890,8 @@ def get_desired_chunk_size(total_size):
   def expand(self, pbegin):
     if isinstance(self.source, BoundedSource):
       return pbegin | _SDFBoundedSourceWrapper(self.source)
+    elif isinstance(self.source, ptransform.PTransform):
+      return pbegin.pipeline | self.source
```

Comment:
This looks pretty awkward but I get the idea. Please add a comment here explaining this.

Reply:
Done.